db/migrate/20151218011129_create_friendly_id_slugs.rb

```diff
@@ -0,0 +1,15 @@
+class CreateFriendlyIdSlugs < ActiveRecord::Migration
+  def change
+    create_table :friendly_id_slugs do |t|
+      t.string   :slug,           :null => false
+      t.integer  :sluggable_id,   :null => false
+      t.string   :sluggable_type, :limit => 50
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Put one space between the method name and the first argument.